Hosts/FunctionsHost/Transport/ResponseSender.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Azure.EventHubs;
+using Microsoft.Extensions.Logging;
+using ReactiveMachine;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace FunctionsHost
+{
+    internal class ResponseSender : BatchSender<IResponseMessage>
+    {
+        public const int NumberPartitions = 32;
```

Comment:
Should this be a configuration parameter?

Reply:
Actually, I will leave it fixed for now. It is not safe for users to change this dynamically, and I think there is no harm in keeping the partitions at the max.